services/client/services.go

```diff
@@ -299,6 +306,14 @@ func (c *client) Watch(sid services.ServiceID, opts services.QueryOptions) (xwat
 	return w, err
 }
 
+func (c *client) HeartbeatService(zone string) (services.HeartbeatService, error) {
+	if zone == "" {
```

Comment:
nit: this case is probably fine, if getHeartbeatService() can create a valid store for zone "", then we should just return it